src/lib/encode/repository/repository.go

```diff
@@ -1,7 +1,6 @@
 package repository
 
 import "net/url"
-
```

Comment:
same comment as this one:
* https://github.com/goharbor/harbor/pull/17131/files#r923010380